src/app/view/components/dot-persona-selector/dot-persona-selector.component.spec.ts

```diff
@@ -31,7 +33,8 @@ describe('DotPersonaSelectorComponent', () => {
                 {
                     provide: DotMessageService,
                     useValue: messageServiceMock
-                }
+                },
+                StringPixels
```

Comment:
we should create a mock of StringPixels here

Reply:
Why? isn't this just a static method, no requests or anything weird that we need to mock, I think.